lib/resource/AuthenticationResult.js

```diff
@@ -34,16 +44,18 @@ AuthenticationResult.prototype.getAccessToken = function getAccessToken(jwt) {
 };
 
 AuthenticationResult.prototype.getAccessTokenResponse = function getAccessTokenResponse(jwt) {
-
   jwt = jwt || this.getJwt();
+
   var resp = {
     'access_token': jwt.compact(),
     'token_type': 'Bearer',
-    'expires_in': jwt.ttl
+    'expires_in': jwt.body.exp
```

Comment:
The `expires_in` field is not an absolute date, it is the lifetime of the token in seconds.  As such, this should be the original `ttl` value that was provided to `authenticateApiRequest`.  See https://tools.ietf.org/html/rfc6749#section-5.1